css-conditional-5/font-tech-format-explainer.md

```diff
@@ -0,0 +1,87 @@
+# `font-tech()` and `font-format()` `<supports-feature>` Extensions to CSS `@supports`
+
+## Authors
+
+* Dominik Röttsches, [@drott](drott@chromium.org)
+* Chris Lilley, [@svgeesus](chris@w3.org)
+* Jonathan Kew, [@jfkthame](jfkthame@gmail.com)
+
+## Introduction
+
+`font-tech()` and `font-format()` are extensions to the CSS Conditional Syntax
+which enable declarative syntax to use a different set of style rules depending
+on font format support, as well as programmatic feature detection of
+capabilities of the font stack. The latter covers a lack of functionality in web
+platform APIs, as most JS APIs and DOM functionality can be easily feature
+detected, but the abilities of the font stack could not be queried in that way
+so far.
+
+The proposed syntax was introduced following discussion in [TAG review
+#666](https://github.com/w3ctag/design-reviews/issues/666) on a previous syntax
+for the `src:` descriptor, and subsequent [resolutions in the CSS
+WG](https://github.com/w3c/csswg-drafts/issues/6520#issuecomment-947810568) to
+harmonize functionality between the `src:` descriptor parsing, and CSS
+Conditional behavior with respect to font feature detection. This had been a
+[request of the
+TAG](https://github.com/w3ctag/design-reviews/issues/666#issuecomment-901220221)
+review.
+
+## Use cases
+
+Background: The use cases listed below for are very similar to the `tech()` and
```

Comment:
Should "for are very similar to" be "are very similar to use cases for" or similar?